src/go/k8s/kuttl-test.yaml

```diff
@@ -8,10 +8,11 @@ kindContainers:
 testDirs:
   - ./tests/e2e
 kindConfig: ./kind.yaml
+kindNodeCache: false
 commands:
   - command: "./hack/install-cert-manager.sh"
   - command: "make deploy"
   - command: "./hack/wait-for-webhook-ready.sh"
 artifactsDir: tests/_e2e_artifacts
 timeout: 300
-parallel: 1
+parallel: 3
```

Comment:
since the VMs for GHA only have two cores, it might be better to leave it at 1? Or does this actually help?